velox/docs/functions/spark/decimal.rst

```diff
@@ -27,14 +27,28 @@ Multiplication
 
 Division
 --------
-
+When config ``SparkRegistrationConfig.allowPrecisionLoss`` is set to true as default.
```

Comment:
Since the description at L5 only covers part of case now, would you add some introduction for allowPrecisionLoss being false there?

Reply:
For add, subtract, multiply, the precision and scale computation is same, only adjust or bounded them.
But for divide, the precision and scale computation is different, so I only add for divide.
And document bounded behavior in the below.